### DISCUSSION TITLE
Firewall requirements for plugin installation

### DISCUSSION THREAD

**Discussion by l3ender** [Genuitec/Copilot4Eclipse-community#3] [Help] [answered]
Hello,

We are wanting to test Copilot4Eclipse and are wanting to know what hostname/firewall requirements there are for installing the plugin in Eclipse from the marketplace. Neither the Copilot4Eclipse [Prerequisites](https://www.genuitec.com/products/copilot4eclipse/docs/prereqs/prerequisites), [Installation](https://www.genuitec.com/products/copilot4eclipse/docs/installation/), or [Firewall Settings](https://www.genuitec.com/products/copilot4eclipse/docs/resources/technotes/firewall-settings/) doc pages mention detail; the firewall settings shows hostnames for communication made by the plugin after installation, but not is what is needed for/during installation.

From what I can tell, Eclipse uses at least https://downloads.genuitec.com during plugin installation but I am not sure if there are others and it would be great if they could be updated/documented accordingly.

Thank you!

**Comment by brianvfernandes** [comment, marked as answer]
Good point - we'll update our documentation shortly, but in the meanwhile, here are the domains you'd need to unblock for installation:

1. `www.genuitec.com`
2. `downloads.genuitec.com`
3. `download.eclipse.org`

**Reply by l3ender** [reply]
Thanks!
